target/linux/apm821xx/config-6.6

```diff
@@ -169,9 +169,7 @@ CONFIG_NR_CPUS=1
 CONFIG_NR_IRQS=512
 CONFIG_NVMEM=y
 CONFIG_NVMEM_LAYOUTS=y
-CONFIG_NVMEM_LAYOUT_U_BOOT_ENV=y
 CONFIG_NVMEM_SYSFS=y
-CONFIG_NVMEM_U_BOOT_ENV=y
 CONFIG_OF=y
 CONFIG_OF_ADDRESS=y
 CONFIG_OF_EARLY_FLATTREE=y
```

Comment:
Why do you remove them here?
If this was accidentially also make sure they are activated in the 6.12 config.

Reply:
where is this diff from?

https://patch-diff.githubusercontent.com/raw/openwrt/openwrt/pull/18798.patch

does not contain anything about env

Reply:
OK so it's related to

https://github.com/openwrt/openwrt/pull/16618

Reverted removal here.